templates/nasa_nex-gddp-cmip6_processing.py

```diff
@@ -0,0 +1,41 @@
+from pathlib import Path
+
+from miranda import convert, io
+
+
+def main():
+    path_nex_out = Path("/jarre/scenario/jlavoie/data/NEX-GDDP-CMIP6_downloaded/")
+    # put all nc that should be in 1 zarr together
+    nex_files = convert.gather_nex(path_nex_out)
+
+    for path, list_files in nex_files.items():
+        # open as dataset
+        ds = convert.dataset_conversion(
+            list_files,
+            add_version_hashes=False,
+            preprocess=False,
+            project="NEX-GDDP-CMIP6",
+        )
+        # the path is already ok, so just reuse it
+        end_path = path.split("downloaded/")[1]
+
+        # get right chunks
+        calendar, size = io._rechunk.get_time_attrs(ds)
+        target_chunks = io._rechunk.translate_time_chunk(
+            {"time": "4 years", "lat": 50, "lon": 50}, calendar, size
+        )
```

Comment:
Does `write_dataset` not call these internally? Maybe we want to expose these to users?

Reply:
now it does!